app/views/users/edit.html.erb

```diff
@@ -27,5 +27,5 @@
 <% if current_user.administrator? %>
 <br> <br><hr><br>
 <h3> Admin Only</h3>
-  <%= link_to raw('<span class="btn">Delete User</span>'), user_path(@user), method: :delete, confirm: "Are you sure!?" %>
+  <%= link_to raw('<span class="btn">Delete User</span>'), user_path(@user), method: :delete, data: {confirm: "Are you sure!?"} %>
```

Comment:
`jquery-ujs` requires us to set `data-confirm` on the element, so the current code is wrong and results in no confirmation before user deletion.